django/forms/extras/__init__.py

```diff
@@ -1,3 +1,12 @@
-from django.forms.extras.widgets import SelectDateWidget
+import warnings
+
+from django.forms.widgets import SelectDateWidget  # NOQA
+from django.utils.deprecation import RemovedInDjango21Warning
 
 __all__ = ['SelectDateWidget']
+
+
+warnings.warn(
+    "django.forms.extras is deprecated. You can find "
+    "SelectDateWidget in django.forms.widgets instead.",
+    RemovedInDjango21Warning, stacklevel=2)
```

Comment:
This usage looks a bit magic to me, but I think it can stay as it is. The module level deprecation warnings in https://github.com/django/django/commit/f59fd15c4928caf3dfcbd50f6ab47be409a43b01 are different than this, so stacklevel=2 can be removed in the following locations:

* https://github.com/django/django/commit/f59fd15c4928caf3dfcbd50f6ab47be409a43b01#diff-9e264d0b47bfdd60be1698bca9bae281R19
* https://github.com/django/django/commit/f59fd15c4928caf3dfcbd50f6ab47be409a43b01#diff-68395a4996a48dd1b3cd34ddd9efe762R12